src/screens/KeyManagement/OCR2KeysCreate.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import TextField from '@material-ui/core/TextField'
+import MenuItem from '@material-ui/core/MenuItem'
+import { ConfirmationDialog } from 'components/Dialogs/ConfirmationDialog'
+import { useOCR2KeyFamilyQuery } from 'hooks/queries/useOCR2KeysQuery'
+
+export interface Props {
+  showCreateKeyDialog: boolean
+  setToggleCreateKeyDialog: (toggleCreateKeyDialog: boolean) => void
+  onCreate: (chainType: string) => Promise<any>
```

Comment:
```suggestion
  onCreate: (chainType: string) => Promise<void>
```

Shouldn't this be void?

Reply:
Changed in [`fb76488` (#38)](https://github.com/smartcontractkit/operator-ui/pull/38/commits/fb764887332acfbae09f46da22d56353c5428069)